.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -32,18 +32,18 @@ If you don't need some of the steps, cross them by removing the "[ ]" and surrou
  - [ ] [QA] Regression test execution
  - [ ] [QA] QA approval
  - [ ] [DIS] Upload release APK and bundle to internal ownCloud instance
- - [ ] [COM] Ping in #android-dev-internal that we are close to sign the new tags (@mmattel)
+ - [ ] [COM] Ping @mmattel that we are close to sign the new tags
  - [ ] [DIS] Upload and publish release bundle and changelog in Play Store
  - [ ] [DIS] Update screenshots in Play Store/GitHub/F-Droid
  - [ ] [GIT] Create and sign tag `vM.m.p` in HEAD commit of release branch, in owncloud/android
  - [ ] [GIT] Move tag `latest` pointing the same commit as the release commit
  - [ ] [DIS] Publish a new [release](https://github.com/owncloud/android/releases) in owncloud/android
  - [ ] [DIS] Release published in Play Store
  - [ ] [COM] Publish post in central.owncloud.org ([Category:News + Tag:android](https://central.owncloud.org/tags/c/news/5/android))
- - [ ] [COM] Inform in #general that release is out
```

Comment:
This is still a communication channel with interested users, so in addition to inform in the "ownCloud General" internal channel, we can still inform here. We usually just tell that a new release is out and paste the link to the extended post in central, so it shouldn't take much time (and we can have the same message for "ownCloud General" 😀)